src/core/stdcpp/vector.d

```diff
@@ -651,48 +654,105 @@ extern(D):
             pointer _First;
             pointer _Last;
         }
-        pointer _Utransfer(bool _move, bool _ifNothrow = false)(pointer _First, pointer _Last, pointer _Dest)
+        static pointer _Utransfer(bool _move, bool _ifNothrow = false)(pointer _First, pointer _Last, pointer _Dest)
```

Comment:
It can be static, but I'm not happy to change the implementation from Microsoft's implementation. It should remain identical, at least for the time being.

Reply:
To be clear, I did leave those TODO's here, because I think they're valid improvements... but I don't think we have unit tests that tests comprehensively enough to detect breakages that we might introduce by deviating from the reference implementation. I wanted to be confident our tests are very robust before making speculative optimisations that deviate from the C++ implementation.

Reply:
That males sense.